src/api/mockBlogApi.js

```diff
@@ -1,91 +0,0 @@
-import delay from './delay';
```

Comment:
If not, when this merged, I can rebase my save/creation branch, and add the method there...